README.md

```diff
@@ -1,43 +1,419 @@
+RUN apt-get update && apt-get install -y curl python2.7 apt-transport-https lsb-release
+RUN export CLOUD_SDK_REPO="cloud-sdk-$(lsb_release -c -s)" && \
+    echo "deb https://packages.cloud.google.com/apt $CLOUD_SDK_REPO main" | tee -a /etc/apt/sources.list.d/google-cloud-sdk.list
+RUN curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add -
+
+ENV HOME /
+RUN  apt-get update && apt-get install -y google-cloud-sdk
+
+RUN gcloud config set core/disable_usage_reporting true
+RUN gcloud config set component_manager/disable_update_check true
+VOLUME ["/.config"]
+```
+
+### CentOS
+
+```dockerfile
+FROM centos
+
+RUN  echo $'[google-cloud-sdk]\n\
+name=Google Cloud SDK\n\
+baseurl=https://packages.cloud.google.com/yum/repos/cloud-sdk-el7-x86_64\n\
+enabled=1\n\
+gpgcheck=1\n\
+repo_gpgcheck=1\n\
+gpgkey=https://packages.cloud.google.com/yum/doc/yum-key.gpg\n\
+       https://packages.cloud.google.com/yum/doc/rpm-package-key.gpg'\
+        >> /etc/yum.repos.d/google-cloud-sdk.repo 
+
+ENV HOME /
+RUN yum install -y google-cloud-sdk which 
+
+RUN gcloud config set core/disable_usage_reporting true
+RUN gcloud config set component_manager/disable_update_check true
+VOLUME ["/.config"]
+```
+
+### Alpine
+
+```dockerfile
+FROM alpine
+RUN apk --no-cache add curl python bash
+ENV HOME /
+#RUN curl https://sdk.cloud.google.com | bash
+
+ARG CLOUD_SDK_VERSION=154.0.1
+ARG SHA256SUM=b38a272872adcd79e93a87aa1867d4fd36567b40898559a57c1679e048529dea
+
+RUN curl -O https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-${CLOUD_SDK_VERSION}-linux-x86_64.tar.gz
+RUN echo "${SHA256SUM}  google-cloud-sdk-${CLOUD_SDK_VERSION}-linux-x86_64.tar.gz" > google-cloud-sdk-${CLOUD_SDK_VERSION}-linux-x86_64.tar.gz.sha256
+RUN sha256sum -c google-cloud-sdk-${CLOUD_SDK_VERSION}-linux-x86_64.tar.gz.sha256
+RUN tar xzf google-cloud-sdk-${CLOUD_SDK_VERSION}-linux-x86_64.tar.gz 
+
+ENV PATH /google-cloud-sdk/bin:$PATH
+RUN gcloud config set core/disable_usage_reporting true
+RUN gcloud config set component_manager/disable_update_check true
+VOLUME ["/.config"]
+```
+
+The SHA256 checksum is listed on the SDK [documentation page](https://cloud.google.com/sdk/downloads#versioned).
+
+> Note: you can pass in the ARG value of the sdk version and checksum as overrides:
+
+```bash
+docker build --build-arg CLOUD_SDK_VERSION=151.0.1 --build-arg SHA256SUM=26b84898bc7834664f02b713fd73c7787e62827d2d486f58314cdf1f6f6c56bb -t alpine_151 --no-cache .
+```
+
+#### Google Appengine base
+
+The original base image in this repository was based off of 
+
+> FROM gcr.io/google_appengine/base
+
+The full Dockerfile for this base can be found [here](google_appengine_base/Dockerfile) for archival.
+
 ---
 
-[`google/cloud-sdk`](https://index.docker.io/u/google/cloud-sdk/) is a
-[Docker](https://docker.io) image bundling all the components and dependencies
-of the [Google Cloud SDK](https://cloud.google.com/sdk/) including alpha and
-beta components.
+## Usecases
```

Comment:
See my comment above. This section doesn't belong to this repository. Consider moving these to somewhere like cloud.google.com/sdk/docs or cloud.google.com/community/tutorials and we can link to them from here.

Remember this readme file is shown on Docker Hub so it should be **super brief**. Take your time to explore official images on Docker Hub and see how long of a readme they have and what is in their readme.